node_cli/utils/docker_utils.py

```diff
@@ -220,7 +224,15 @@ def get_up_compose_cmd(services):
     return ('docker-compose', '-f', COMPOSE_PATH, 'up', '-d', *services)
 
 
-def compose_up(env):
+def get_up_compose_sync_cmd():
+    return ('docker-compose', '-f', SYNC_COMPOSE_PATH, 'up', '-d')
+
+
+def get_compose_path(sync_node: bool) -> str:
+    return SYNC_COMPOSE_PATH if sync_node else COMPOSE_PATH
+
+
+def compose_up(env, sync_node=False):
```

Comment:
Unused `sync_node=False` here. May be it's better to merge it with `compose_up_sync` instead.